pkg/codegen/docs/gen.go

```diff
@@ -1129,10 +1132,20 @@ func (mod *modContext) genResource(r *schema.Resource) resourceDocArgs {
 		Notes:      mod.pkg.Attribution,
 	}
 
+	var titleTag string
+	if val, ok := titleLookup[mod.pkg.Name]; ok {
+		titleTag = val
+	} else {
+		titleTag = mod.pkg.Name
+	}
+
 	stateParam := name + "State"
+
 	data := resourceDocArgs{
 		Header: header{
-			Title: name,
+			Title:    name,
+			TitleTag: fmt.Sprintf("Resource %s | Package %s", name, titleTag),
+			MetaDesc: regexp.MustCompile(`(?m)^.*$`).FindString((r.Comment)),
```

Comment:
Let's pull this regexp to be a file-level `var` instead of compiling for each resource. It would also be helpful to have a comment to indicate what the regexp is trying to do.

Reply:
ahh.. good point. done.